v/vault-k8s/vault-k8s-1.2.1-ubi-8.7.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash -ex
+# ----------------------------------------------------------------------------
+#
+# Package       : vault-k8s
+# Version       : 1.2.1
+# Source repo   : https://github.com/hashicorp/vault-k8s
+# Tested on     : UBI 8.7
+# Language      : Go 
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Sumit Dubey <Sumit.Dubey2@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=vault-k8s
+PACKAGE_VERSION=${1:-1.2.1}
+PACKAGE_URL=https://github.com/hashicorp/vault-k8s
+
+#Install dependencies
+dnf install -y \
+    gcc \
+    gcc-c++ \
+    git \
+    make \
+    wget 
+
+wdir=`pwd`
+
+#Download source code
+cd $wdir
+git clone ${PACKAGE_URL}
+cd ${PACKAGE_NAME} && git checkout v${PACKAGE_VERSION}
+
+#Install Golang
+cd $wdir
+wget https://go.dev/dl/go1.20.5.linux-ppc64le.tar.gz
+rm -rf /usr/local/go && tar -C /usr/local -xzf go1.20.5.linux-ppc64le.tar.gz
+rm -rf go1.20.5.linux-ppc64le.tar.gz
+export PATH=$PATH:/usr/local/go/bin
+go version
+export PATH=$PATH:$HOME/go/bin
+
+#Install docker
+yum remove -y docker \
```

Comment:
I'd feel more comfortable not removing any packages from the customer's environment if it can be avoided, maybe we can assume the script will run in a clean env. and put comments if we anticipate conflicts with already installed packages?

Reply:
done